Workbooks/SapMonitor2.0/AIOpsInsights/Availability Insights/Auto RCA.workbook

```diff
@@ -230,21 +281,9 @@
           "comparison": "isNotEqualTo"
         }
       ],
-      "customWidth": "50",
+      "customWidth": "0",
```

Comment:
FYI
`customeWidth` is zero

Reply:
This is intentional. Will ensure width shrinks when content is less.